examples/python-text2img/handler.py

```diff
@@ -0,0 +1,23 @@
+import requests
+import os
+from bottle import HTTPResponse
+
+def main(event, context):
+
+    text = event["extensions"]["request"].params.get("text")
+
+    if text:
+        url = os.getenv('URL')
+        apiKey = os.getenv('API-KEY')
+
+        print('Trying to query {} with "{}"'.format(url, text))
+
+        # Send request to the external service
+        response = requests.post(url, files=dict(text=text), headers={"api-key": apiKey})
```

Comment:
I prefer consistency - in this case in creating dictionary. We could write it as:
```
response = requests.post(url, files={'text'=text}, headers={'api-key': apiKey})
```

Reply:
somehow the proposed change does not compile.

Reply:
My mistake. 
```
response = requests.post(url, files={'text': text}, headers={'api-key': apiKey})
```

Reply:
done